io/module_wrt_grid_comp.F90

```diff
@@ -160,6 +160,9 @@ subroutine wrt_initialize(wrt_comp, imp_state_write, exp_state_write, clock, rc)
       type(ESMF_Array)                        :: array_work, array
       type(ESMF_FieldBundle)                  :: fieldbdl_work
       type(ESMF_Field)                        :: field_work, field
+! CHJ --- Gerhard's update: quilting with non-factors for layout --- s ---
+      type(ESMF_Decomp_Flag)                  :: decompflagPTile(2,6)
+! CHJ --- Gerhard's update --- e ----------
```

Comment:
Please remove all comment lines starting with `! CHJ --- Gerhard's update`

Reply:
This is the same change request that @mkavulich mentioned for the corresponding PR to release/public-v2. I can make the changes for release/public-v2 (because I pulled this PR into my fv3atm PR), but then it would be good if @chan-hoo could cherry-pick my commit. Or vice versa, I wait for @chan-hoo to remove those comments and cherry-pick it into the release/public-v2 PRs.

Reply:
@DusanJovic-NOAA @climbfuji  Removed!

Reply:
Chan-hoo, the comments "! CHJ --- Gerhard's update" are still here.

Reply:
@junwang-noaa , Something strange. I can't see the inline comments in my branch. I don't know why they are still shown in the above window. However, on the 'file changed' tab (line 163), they are not found either.